Fixes #242 Added images in entertainment skills

madhavrathi opened the pull request:
Fixes issue #242 

Changes: Added images in skills of entertainment model
@DravitLochan @dynamitechetan @saurabhjn76 @AliAyub007 Please review.
Thanks

mariobehling reviewed (changes requested):
Images should be stored with the skill itself, not on an external website.

DravitLochan reviewed (changes requested):
The link of the image has to be from susi_skill_data repo

mariobehling commented:
@madhavrathi Please follow up.

madhavrathi commented:
@mariobehling I will add images in the repo today. Thanks

mariobehling commented:
@madhavrathi Not sure, what is taking time to add a few images. Let's speed up, please.

madhavrathi commented:
@mariobehling I was finishing up a PR in chat.susi.ai repo. Will add images right now and resolve conflicts asap. 
Sorry for the delay. will speed up my work

madhavrathi commented:
@mariobehling @DravitLochan I have added the images with a similar name as the skill to the repo itself. If we want to proceed with links to images from this repo, then they need to be merged first. Then I will be able to access them using raw githubusercontent. Please guide with any other method, if any.

DravitLochan reviewed (changes requested):
image for f1_racing is not suitable.

madhavrathi commented:
@DravitLochan Please review again. 
Thanks

mariobehling reviewed (changes requested):
Images need to go into a subfolder below  "models/general/entertainment/en/images". You need to use relative paths instead of absolute. This is basic knowledge. Of course the repo needs to work, when I deploy it elsewhere. It is sad to see with how little attention to detail you are working here. Such PRs and requirements to get lots of reviews slow down the project. This kind of low quality PR is not sufficient to pass the third term in GSoC.

mariobehling commented on models/general/entertainment/en/images.txt:
```diff
@@ -1,4 +1,5 @@
 images of *|pictures of *|get me images on *|How does  * looks like|show me a picture of *|a picture of *|an image of *|show me a *|can you show me a *|visuals on *|visuals of *|image of *|picture of *|how would * look like|get me a glipmse of *|pictorial representation of *|fetch me some *|diagram of *
+!image: https://raw.githubusercontent.com/fossasia/susi_skill_data/master/models/general/entertainment/en/images.jpg
```
Why is this image named "images.jpg"? Images should have the same name as the skill.

Just like many other images, the image here is not relevant for the skill.

All images need to be in a square format. So that they fit in easily into the cms. Please do compare your work with how images are used in Alexa (https://www.amazon.com/b?node=13727921011)

mariobehling commented:
Many images are unrelated to the skill. Images need to be in square format and need to have the same name as a skill.

This PR is taken care of in a very superficial way. Expectations to a student in the third term are much higher. What is going on with you?

mariobehling commented:
@dynamitechetan How is it possible that you accepted this PR?

madhavrathi commented:
@mariobehling I am very sorry you feel this way. I will improve this PR in the way expected asap. Sorry for the trouble.

madhavrathi commented:
@mariobehling regarding the image with name images.jpg there is a skill named images, so I thought this name is appropriate. I have given all tje images same name as skill. I will make them square and add more relevant images.

madhavrathi commented:
@mariobehling I have added the relative path and also changed few images.All the images are now square. Extremely sorry for the delay and discussion in this PR. I will try to speed up things and match the expectations of third evaluations. Please comment if any image needs to be changed, I will do it.
Thanks

mariobehling commented on models/general/entertainment/en/cricket.txt:
```diff
@@ -1,4 +1,6 @@
-matches going on|current matches|Do you know about matches going on?|Do you know cricket matches going on?|Tell me about matches going on?|Tell me about current cricket matches|Do you know about matches going on|Do you know cricket matches going on|is there any cricket match today?|what are the cricket matches going on?|is there any cricket match today|what are the cricket matches going on|find c ...
+
```
Why is this empty line added?

mariobehling reviewed (changes requested):
I cannot believe I am asking this to a developer in the third term of GSoC: Please research the Internet how to scale an image. The ratio is out of proportion in many images.

madhavrathi commented:
@mariobehling Scaling of previous images was cropping off the content of the image. I have now added new images and the ratio is in proportion for all the images. Please review
I apologize for the trouble.

madhavrathi commented:
@mariobehling I have also added the attribute to flat icons website in the comment section in skills. Their website policy requires this for free use.

mariobehling commented on models/general/entertainment/en/cat.txt:
```diff
@@ -1,6 +1,7 @@
 # tells fun fact about cats
-
-tell me a cat fact | tell something interesting about cat | Do you know any amazing cat fact | give me a cat fact | do you know any cat fact | tell me facts about cat | Do you know anything amazing about cats | cats fact | tell me fun trivia about cat| fun trivia about cat | cat fun trivia | What is so special about cats?| tell me awesome fact about cats| tell me awesome cat facts| awesome cat fa ...
+#Icon made by Freepik from www.flaticon.com
```
Please take this out and only use images that don't require this kind of note.

mariobehling commented:
Please take out the flaticons.com spam and images from there if this is requested. There are tons of freely licensed images on the web that don't require this, e.g. https://openclipart.org and https://pixabay.com.

Please finalize this PR by getting it up to standard and let us reduce the time that is required to follow up.

madhavrathi committed:
Fixes #242 Added images in entertainment skills

madhavrathi commented:
@mariobehling Added the new images and removed the flaticons images. Please review
